champions/src/main/java/me/mykindos/betterpvp/champions/weapons/ChampionsWeaponManager.java

```diff
@@ -0,0 +1,29 @@
+package me.mykindos.betterpvp.champions.weapons;
+
+import com.google.inject.Inject;
+import me.mykindos.betterpvp.champions.Champions;
+import me.mykindos.betterpvp.core.combat.weapon.Weapon;
+import me.mykindos.betterpvp.core.combat.weapon.WeaponManager;
+import me.mykindos.betterpvp.core.framework.adapter.Adapters;
+import me.mykindos.betterpvp.core.framework.adapter.PluginAdapter;
+import org.reflections.Reflections;
+
+@PluginAdapter("Core")
```

Comment:
Not sure why we need a PluginAdapter for core? seeing as all plugins are hard-dependent on it.

Other than that, this class just seems like a loader similar ChampionsCommandLoader & ChampionsListenerLoader

Reply:
i only did that so it gets loaded without me having to call its constructor, how do you want me to load it